config/environments/production.rb

```diff
@@ -17,13 +17,13 @@
   config.action_mailer.default_url_options = { host: "diaper.app" }
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.smtp_settings = {
-    :address        => 'smtp.sendgrid.net',
-    :port           => '587',
-    :authentication => :plain,
-    :user_name => ENV['SENDGRID_USERNAME'],
-    :password => ENV['SENDGRID_PASSWORD'],
-    :domain         => 'diaper.app',
-    :enable_starttls_auto => true
+    address: 'smtp.sendgrid.net',
```

Comment:
oh lol, I ended up fixing these in https://github.com/rubyforgood/diaper/pull/2215